ui/src/components/DataTable/index.jsx

```diff
@@ -132,12 +144,11 @@ export default class DataTable extends Component {
               <TableHead>
                 <TableRow>
                   {headers.map(header => (
-                    <TableCell key={`table-header-${header.label}`}>
+                    <TableCell key={`table-header-${header.id}`}>
                       <TableSortLabel
-                        id={header.label}
-                        active={header.label === sortByHeader}
+                        active={header.id === sortByLabel}
                         direction={sortDirection || 'desc'}
-                        onClick={this.handleHeaderClick}>
+                        onClick={() => this.handleHeaderClick(header)}>
```

Comment:
Hm, I think this looks a bit different than the the diff.

Reply:
Yes, the reason for that is because when I call the function the way it is in the diff, react throws an error `Uncaught Invariant Violation: Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.` so I had to pass a reference of the `handleHeaderClick` function to avoid it being repeatedly called when `render()` is called.

Reply:
Interesting. Can you try again, please? I wonder if it was because it needed a hard refresh. It seems to work on my end.

Reply:
Ok will do

Reply:
I did a refresh for this but I still got the same error